sapi/fpm/fpm/fpm_log.c

```diff
@@ -206,8 +206,8 @@ int fpm_log_write(char *log_format) /* {{{ */
 							len2 = snprintf(b, FPM_LOG_BUFFER - len, "%.3f", proc.duration.tv_sec + proc.duration.tv_usec / 1000000.);
 						}
 
-					/* miliseconds */
-					} else if (!strcasecmp(format, "miliseconds") || !strcasecmp(format, "mili")) {
+					/* milliseconds */
+					} else if (!strcasecmp(format, "milliseconds") || !strcasecmp(format, "mili")) {
```

Comment:
This one looks BC breaking.

Reply:
Wasn't this something recently introduced with the new FPM logger? cc @bukka

Reply:
@Kalle No, it has been introduced with bbe8518456eb883073f44f643a833843c4743410 (available as of PHP 5.3.11).

Reply:
@cmb69 thanks for the swift checking, in that case I stand with @nikic

Reply:
We should however add a fall back to the correct spelling, leaving the incorrect one until the next major or similar